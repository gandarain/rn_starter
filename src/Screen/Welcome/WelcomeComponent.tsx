import React from 'react';
import { Text } from 'react-native';

import type { Props } from './types';

/**
 * WelcomeScreen
 * @returns {React.FC<Props>} - Component
 */
const WelcomeScreen: React.FC<Props> = () => <Text>Hello</Text>;

export default WelcomeScreen;
