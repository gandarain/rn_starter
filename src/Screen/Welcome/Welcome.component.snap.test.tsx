import * as React from 'react';
import renderer from 'react-test-renderer';

import WelcomeComponent from './Welcome.component';

jest.mock('../../Component', () => ({
  GeneralText: 'GeneralText',
  Wrapper: 'Wrapper',
  Button: 'Button',
}));

describe('<WelcomeComponent />', () => {
  it('Should render WelcomeComponent', () => {
    const tree = renderer.create(<WelcomeComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
