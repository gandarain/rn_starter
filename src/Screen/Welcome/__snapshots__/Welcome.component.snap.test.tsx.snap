// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeComponent /> Should render WelcomeComponent 1`] = `
<Wrapper>
  <Text
    style={
      Object {
        "color": "#1c1c1c",
        "fontFamily": "Roboto-Bold",
        "fontSize": 20,
      }
    }
  >
    Hello
  </Text>
  <GeneralText
    color="#1c1c1c"
    size={25}
    title="Hello"
    variant="Roboto-Bold"
  />
  <Icon
    color="#ea3d13"
    name="rocket"
    size={25}
  />
</Wrapper>
`;
